internal/controller/csiaddons/csiaddonsnode_controller.go

```diff
@@ -271,3 +273,21 @@ func validateCSIAddonsNodeSpec(csiaddonsnode *csiaddonsv1alpha1.CSIAddonsNode) e
 
 	return nil
 }
+
+// parseCapabilities returns a list of capabilities in the format
+// capability.Type
```

Comment:
can you add a little note/example of how a `cap.String()` looks like, and what the desired output for it is?

Reply:
Done